VERSIONING.md

```diff
@@ -6,16 +6,17 @@ For context on how the CNB project thinks about versioning, see [RFC#10](https:/
 [spec]: #spec
 The spec describes two components that are [separately versioned](https://github.com/buildpacks/spec#api-versions). These APIs are versioned as follows:
 
+The APIs are denoted by two numbers "`<major>`.`<minor>`".
+
 #### Platform API Version
 Given by CNB platforms to the lifecycle, this version indicates which platform API versions that platform currently support. In review, in [PR#67](https://github.com/buildpacks/spec/pull/67/files), this version indicates the compatibility between a CNB platform and a given lifecycle according to the following rules:
-- When `<major>` is `0`, the platform is only compatible with lifecycles implementing that exact Platform API.
-- When `<major>` is greater than `0`, the platforms is only compatible with lifecycles implementing platform API
-`<major>.<minor>`, where `<major>` of the lifecycle equals
+- For Platform API versions `0.1` - `0.2`, the platform is only compatible with lifecycles implementing that exact Platform API.
+- For Platform API versions greater than or equal to `0.3`, the platform is guaranteed to be compatible with any lifecycle which supports a platform API greater than or equal to that requested by the platform. The platform must set `CNB_PLATFORM_API` in the lifecycle’s environment to configure this behavior.
```

Comment:
I don't think this PR is valid anymore since https://github.com/buildpacks/rfcs/blob/main/text/0049-multi-api-lifecycle-descriptor.md. Now the lifecycle supports multiple platform APIs and lists all supported APIs in the lifecycle descriptor file and `io.buildpacks.lifecycle.apis` label. For `0.x` API the lifecycle must explicitly indicate support for the desired API.